README.md

```diff
@@ -67,6 +67,25 @@ local IsVisible = RegionState:IsRegionVisible("Region1") --Bool for if a region
 local InRegion = RegionState:IsInRegion("Region1", Vector3.new()) --Bool for if a point is in a region
 ```
 
+## Quicker Region Setup
+`RegionState` offers an additional method that makes it easier to create a centralized folder of "Regions" that you can easily customize, expand, rotate, resize, and re-position. 
+```
+Regions : Instance
+|- Region1 : Instance
+|  |-- ZonePart : BasePart
+|  |-- ZonePart : BasePart
+|- Region2 : Instance
+|  |-- ZonePart : BasePart
+```
+Using the instance above as an example,
+```lua
+local RegionCulling = require(game:GetService("ReplicatedStorage"):WaitForChild("RegionCulling"))
+local RegionState = RegionCulling.RegionState
+
+RegionState:InsertRegionsFromInstance(game.ServerStorage.Regions)
```

Comment:
Everything runs on the client. Referencing `ServerStorage` will be... problematic. Also should use `GetService` to be consistent with the other examples.